templates/settings.php

```diff
@@ -237,6 +237,35 @@
 			<?php \do_settings_fields( 'activitypub', 'general' ); ?>
 			<?php \do_settings_fields( 'activitypub', 'server' ); ?>
 		</div>
+		<div class="box">
+			<h3><?php \esc_html_e( 'Security', 'activitypub' ); ?></h3>
+			<table class="form-table">
+				<tbody>
+					<?php if ( ! defined( ACTIVITYPUB_AUTHORIZED_FETCH ) ) : ?>
+					<tr>
+						<th scope="row">
+							<?php \esc_html_e( 'Authorized-Fetch', 'activitypub' ); ?>
+						</th>
+						<td>
+							<p>
+								<label>
+									<input type="checkbox" name="activitypub_authorized_fetch" id="activitypub_authorized_fetch" value="1" <?php echo \checked( '1', \get_option( 'activitypub_authorized_fetch', '0' ) ); ?> />
```

Comment:
```suggestion
									<input type="checkbox" name="activitypub_authorized_fetch" id="activitypub_authorized_fetch" value="1" <?php \checked( '1', \get_option( 'activitypub_authorized_fetch', '0' ) ); ?> />
```
`checked()` echos automatically, unless the third parameter is set to `false`.